### DISCUSSION TITLE
uninstalling neovim

### DISCUSSION THREAD

**Discussion by johnlange2** [community/community#120133] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

I installed neovim on my Mac, but now I need to remove it. What is the procedure for uninstalling it? (This should also be added to the README.md file here https://github.com/github/copilot.vim/README.md)

**Comment by kalebers** [comment]
Hello!

If you used pip to install it, run first the following command on your terminal:
`pip uninstall neovim`

After that, you can use the command `rm -rf ~/.config/nvim` to remove configuration files form NeoVim.

Now, if you used homebrew to install it, follow the same command as mentioned before, changing pip for brew, as:
`brew uninstall neovim`

An then run the command `rm -rf ~/.config/nvim` to remove configuration files. 

The uninstall procedure should be easy, any problems or questions, just let me know!

**Reply by johnlange2** [reply]
Hi @kalebers,
I did not use pip or brew to install neovim. Rather, I used the documented command:
`git clone https://github.com/github/copilot.vim.git \
  ~/.config/nvim/pack/github/start/copilot.vim`

Also, I do not have a folder ~/.config/nvim. I do have a folder ~/.config/github-copilot.
Should I remove that folder in order to uninstall neovim?

**Reply by kalebers** [reply]
Hi again! 

Seems that you installed Neovim in the directory for the GitHub copilot. 

If you don't have anything that you still want to use, in order to uninstall it, you should remove all the files in that path. Running the command `rm -rf ~/.config/github-copilot` should do the trick to delete everything.

Or use `find ~/.config/github-copilot -name "*nvim*" -exec rm -rf {} \;` to look just for neovim files in the given path and delete only the files related to it.

Any doubts, you know where to find us!

**Reply by johnlange2** [reply]
Hi @kalebers,

I didn't intentionally install Neovim in ~/.config/github-copilot. To install it, I just followed the instructions on the page here https://github.com/github/copilot.vim/README.md.

I have no files with nvim in my github-copilot folder:
```
jlange2@TQ7WVPVG3XT:~/.config% ls
SEGGER         github-copilot iterm2         wireshark
jlange2@TQ7WVPVG3XT:~/.config%
jlange2@TQ7WVPVG3XT:~/.config% ls github-copilot
hosts.json    versions.json
```

To verify that Copilot is still installed (even though my Copilot subscription is deactivated), the :Copilot command in vi gives this:
```
Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as johnlange2.

Request Actions:
1. Copilot Settings
2. Dismiss
Type number and <Enter> or click with the mouse (q or empty cancels): Not authenticated: NotAuthorized
```

After removing the github-copilot folder, the :Copilot command in vi gives this:
```
First copy your one-time code: 1D09-C4C6
Press ENTER to open GitHub in your browser
```

So Copilot seems to still be installed. What I'm looking for is how to totally remove Copilot from my system.

Thanks,
-John

**Reply by johnlange2** [reply, marked as answer]
Just to finish off this topic, I had installed Copilot with this command:
```
git clone https://github.com/github/copilot.vim.git \
  ~/.vim/pack/github/start/copilot.vim
```

To remove Copilot completely, I just removed the folder where I had cloned it:
`rm -rf ~/.vim/pack/github/start/copilot.vim`
